src/main/asciidoc/_chapters/upgrading.adoc

```diff
@@ -125,10 +125,10 @@ for warning about incompatible changes). All effort will be made to provide a de
 [[hbase.client.api.surface]]
 ==== HBase API Surface
 
-HBase has a lot of API points, but for the compatibility matrix above, we differentiate between Client API, Limited Private API, and Private API. HBase uses a version of link:https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/Compatibility.html[Hadoop's Interface classification]. HBase's Interface classification classes can be found link:https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/classification/package-summary.html[here].
+HBase has a lot of API points, but for the compatibility matrix above, we differentiate between Client API, Limited Private API, and Private API. HBase uses link:http://yetus.apache.org/documentation/0.5.0/interface-classification/[Apache Yetus Audience Annotations] to guide downstream expectations for stability. In previous releases HBase relied on its own implementation of these classes; folks can rely on the same assumptions based on the annotation class name, but the package will be different.
```

Comment:
I suggest breaking out Client API, Limited Private API, and Private API into a little table or list with definitions of each.

The last sentence is kind of inside baseball. I suggest something like "HBase no longer maintains its own implementation of these classes but relies on packages provided by Apache Yetus.

Reply:
a table independent of the explanation in the InterfaceAudience section below? or in place of?

Reply:
In place of.

Reply:
just to confirm. there is a set of sub-sections under here that has brief definitions of each of Client API, LP API, and private API:

http://hbase.apache.org/book.html#hbase.client.api.surface

I'm turning those into a table?

Reply:
It's not that important. I think I just sensed that the info was both confusing and kind of matrix-y.